requirements/prod.txt

```diff
@@ -140,7 +168,7 @@ salesforce-bulk==2.2.0
     # via -r requirements/prod.in
 sarge==0.1.6
     # via -r requirements/prod.in
-selenium==3.141.0
+selenium==4.0.0
```

Comment:
@Br4nd0R Let's not update selenium until @boakley has a chance to evaluate it.

Reply:
Actually we clearly can't update to selenium 4 without further work. For one thing, it no longer supports Python 3.6 (you can see those builds failed). Also all of the robot tests are failing.